src/factory-reset/factory-reset-tool.c

```diff
@@ -316,18 +319,6 @@ static int vl_method_get_factory_reset_mode(sd_varlink *link, sd_json_variant *p
         return sd_varlink_replybo(link, SD_JSON_BUILD_PAIR_STRING("mode", factory_reset_mode_to_string(f)));
 }
 
-static int vl_method_can_request_factory_reset(sd_varlink *link, sd_json_variant *parameters, sd_varlink_method_flags_t flags, void *userdata) {
-        int r;
-
-        assert(parameters);
-
-        r = sd_varlink_dispatch(link, parameters, /* table= */ NULL, /* userdata= */ NULL);
-        if (r != 0)
-                return r;
-
-        return sd_varlink_replybo(link, SD_JSON_BUILD_PAIR_BOOLEAN("supported", is_efi_boot()));
-}
```

Comment:
uh??? sorry, but i am strictly against adding new "apis" for downstream apps that are touch files. touch files are ok for internal stuff, but expose to the outside, nah.

also, the idea was that having IPC for this allows us to flexibly replace/tweak the backing logic for it.